homeassistant/bootstrap.py

```diff
@@ -112,13 +112,7 @@ def from_config_dict(config: Dict[str, Any],
     config = OrderedDict(config)
 
     # Merge packages
-    conf_util.merge_packages_config(
-        hass, config, core_config.get(conf_util.CONF_PACKAGES, {}))
-
-    # Ensure we have no None values after merge
-    for key, value in config.items():
-        if not value:
-            config[key] = {}
```

Comment:
What happened to these lines? I think they're required for components that accept empty configurations

Reply:
I noticed during initial testing that those are actually duplicates and therefore not needed.

Inside `merge_package_config` we already assign `OrderedDict` to keys with empty values: https://github.com/home-assistant/home-assistant/blob/bd09e9668192c958a2c87a19788818afab08af70/homeassistant/config.py#L692-L693

The corresponding test `test_config -> test_merge`: https://github.com/home-assistant/home-assistant/blob/b0ba1c653e0fa542de2529f96130d17e10ce744e/tests/test_config.py#L583